lib/Gruntfile.js

```diff
@@ -0,0 +1,34 @@
+module.exports = function(grunt) {
+  grunt.initConfig({
+  jshint: {
+  files: ['../scripts/**/*.js'], //location of javascript files
+  options: {
+  predef: ["document", "console", "$" ], //allows for predefined things not found in js
+  esnext: true, //allows for ES6
+  globalstrict: true,
+  browser: true,
+  globals: {"CarLot":true} //name value pairs, allows to define global vars used in  files.
+  }
+  },
+  sass: { //setup sass compilation
+  dist: {
+  files: {
+  '../styles/styles.css': '../sass/styles.scss'
```

Comment:
Grunt did not initially run for me because you did not have your sass folder or styles.scss file.